sources/lib/Model/ContainerModelPooler.php

```diff
@@ -10,8 +10,8 @@
 namespace PommProject\PommBundle\Model;
```

Comment:
I don’t undestand the goal of changes in this file. The `ContainerAwareTrait` already requires a `ContainerInterface`.